dashboard.csss-ci.com/server/api/models/Townhalls_model.php

```diff
@@ -1,49 +1,82 @@
 <?php
-
+/**
+checks to see if BASEPATH is defined
+*/
 if (!defined('BASEPATH'))
     exit('No direct script access allowed');
-
+/**
+class that controls Townhall model
+extends from CI_Model
+*/
 class Townhalls_model extends CI_Model {
-
+    //private table for townhall
     protected $table = "townHalls";
-
+    /**
+    function for saving data
+    @param $data which is array or list
```

Comment:
Please follow the format:
/**
* [Description of method]
*
*@param [type] [param name] - [description of param]
*@param [type] [param name] - [description of param]
*@return [type] [description of return]
*/